tensilelite/Tensile/SolutionStructs.py

```diff
@@ -2807,6 +2807,16 @@ def assignDerivedParameters(state):
           state["LdsPadB"] = 0
       assert(state["LdsPadB"] >= 0)
 
+    if state["LdsBlockSizePerPadA"] != 0:
```

Comment:
what will happen if we set LdsBlockSizePerPadA=-1 for odd MIWT0 ?

Reply:
when state["LdsBlockSizePerPadA"] = -1, that will be assigned a value at [SolutionStructs.py#L2671](https://github.com/ROCm/hipBLASLt/blob/develop/tensilelite/Tensile/SolutionStructs.py#L2671)
Hence, at line 2810, there will be a real value (not -1).  then we can use this value to check if the LDS address is consistent.

Reply:
if user set -1 which mean let other kern-genator to produce the most proper setting for their config, then we should generate a least one kernel. If -1 for those odd MT0 would produce a non-zero value at [SolutionStructs.py#L2671](https://github.com/ROCm/hipBLASLt/blob/develop/tensilelite/Tensile/SolutionStructs.py#L2671) and rejected by here, it means -1 doesn't work.
What I mean is that we should give a solution at least for -1 setting.

Reply:
OK, so we can set state["LdsBlockSizePerPadA"] to 0 for the inconsistent case instead of the reject.